c_src/CMakeLists.txt

```diff
@@ -12,6 +12,15 @@ include(ASan)
 IF (APPLE)
   set(CMAKE_MODULE_LINKER_FLAGS "-flat_namespace -undefined suppress")
   set(CMAKE_MACOSX_RPATH 1)
+  if (NOT DEFINED ENV{SDKROOT})
+    execute_process(COMMAND
+      xcrun --show-sdk-path
+      OUTPUT_VARIABLE APPLE_SDK_ROOT
+      )
+  else()
+     set(APPLE_SDK_ROOT $ENV{SDKROOT})
+  endif()
+  set(GF_APPLE_ENV "SDKROOT=${APPLE_SDK_ROOT}")
```

Comment:
Oops. the GF_ naming from the Galois fields library carried over from https://github.com/helium/erlang-erasure/pull/10. It's not a huge deal if you don't feel like fixing it. I can't think of anything better than `APPLE_SDKROOT_ENV`